addons/website_event/tests/common.py

```diff
@@ -73,28 +73,28 @@ def setUpClass(cls):
         )
 
     def _get_menus(self):
-        return set(['Introduction', 'Location', 'Register'])
+        return set(['Introduction', 'Location', 'Register', 'Community'])
 
-    def _assert_website_menus(self, event, menu_entries=None):
+    def _assert_website_menus(self, event, menus_in, menus_out=None):
         self.assertTrue(event.menu_id)
 
-        if menu_entries is None:
-            menu_entries = self._get_menus()
+        if menus_in is None:
+            menus_in = list(self._get_menus())
 
         menus = self.env['website.menu'].search([('parent_id', '=', event.menu_id.id)])
-        self.assertEqual(len(menus), len(menu_entries))
-        self.assertEqual(set(menus.mapped('name')), menu_entries)
+        self.assertTrue(len(menus) >= len(menus_in))
+        self.assertTrue(all(menu_name in menus.mapped('name') for menu_name in menus_in))
+        if menus_out:
```

Comment:
Could init menus_out to `[]` in method parameters as `all([])` returns True.
That way you avoid a if here.

Reply:
Giving a mutable as parameter is dangerous ... would avoid.